ecommerce/api.py

```diff
@@ -188,6 +189,8 @@ def fulfill_completed_order(
     order.fulfill(payment_data)
     order.save()
 
+    add_fulfilled_order_to_paid_courserun(order)
```

Comment:
I'd suggest this should be added to `Order.fulfill()` so that we can be sure if that's called from anywhere else that this logic is always executed.